src/main/groovy/com/ullink/Ikvm.groovy

```diff
@@ -60,12 +61,48 @@ class Ikvm extends ConventionTask {
     
     @InputFile
     def getIkvmc(){
-        assert getIkvmHome(), "You must install Ikvm and set ikvm.home property or IKVM_HOME env variable"
-        File ikvmExec = new File(project.file(getIkvmHome()), 'bin/ikvmc.exe')
+        def home = resolveIkvmHome()
+        assert home, "You must install Ikvm and set ikvm.home property or IKVM_HOME env variable"
+        File ikvmExec = new File(project.file(home), IKVM_EXE)
         assert ikvmExec.exists(), "You must install Ikvm and set ikvm.home property or IKVM_HOME env variable"
         return ikvmExec
     }
 
+    File resolveIkvmHome() {
+        def home = getIkvmHome()
+        URL url
+        if (home?.toString()?.startsWith("http"))
+            url = new URL(home.toString())
```

Comment:
Nothing harmful, but if home's a URL, then this condition may be verified (while only the next one should do)